src/components/UserProfile.jsx

```diff
@@ -174,7 +174,12 @@ export default function UserProfile({
               someoneElse ? 'NO_SIGHTINGS_NON_SELF' : 'NO_SIGHTINGS'
             }
           />
-          {!someoneElse && <CollaborationsCard userId={userId} />}
+          {!someoneElse && (
+            <CollaborationsCard
+              htmlId="collab-card"
```

Comment:
Oh yeah. I forgot that I also propagated the necessary element id attribute down to the material element.